veggieseasons/lib/widgets/search_bar.dart

```diff
@@ -19,45 +21,51 @@ class SearchBar extends StatelessWidget {
   Widget build(BuildContext context) {
     final themeData = CupertinoTheme.of(context);
 
-    return DecoratedBox(
-      decoration: BoxDecoration(
-        color: Styles.searchBackground(themeData),
-        borderRadius: BorderRadius.circular(10),
-      ),
-      child: Padding(
-        padding: const EdgeInsets.symmetric(
-          horizontal: 4,
-          vertical: 8,
-        ),
-        child: Row(
-          children: [
-            ExcludeSemantics(
-              child: Icon(
-                CupertinoIcons.search,
-                color: Styles.searchIconColor,
-              ),
-            ),
-            Expanded(
-              child: CupertinoTextField(
-                controller: controller,
-                focusNode: focusNode,
-                decoration: null,
-                style: Styles.searchText(themeData),
-                cursorColor: Styles.searchCursorColor,
-              ),
+    return ClipRRect(
```

Comment:
Improvement in the ui design so that scrolled list is blurred under the search bar and not visible due to it's opacity.